packages/shared/lib/app.ts

```diff
@@ -137,3 +138,10 @@ export const logout = (_clearActiveProfile: boolean = false, _lockStronghold: bo
             _cleanup()
         }
     })
+
+/**
+ * The privacy policy packaged with the current version of Firefly
+ */
+export const PRIVACY_POLICY_VERSION = 2
```

Comment:
I think it would have to be in a file because this versioning actually doesn't exist outside of Firefly. Basically, we just increment this number the next time we update the privacy policy. It's purely for triggering the "we've updated our privacy policy" alert, if that makes sense